packages/rancher-sriov/generated-changes/patch/values.yaml.patch

```diff
@@ -36,25 +36,25 @@
 -  webhook: ghcr.io/k8snetworkplumbingwg/sriov-network-operator-webhook
 +  operator:
 +    image: rancher/hardened-sriov-network-operator
```

Comment:
We should follow the annotation of `repository: {imagename}` and `tag: {tag}` to make sure that these images are included in rancher-images.txt so that airgap customers can use this chart

Reply:
Same goes for all other charts

Reply:
There is a checkpoint also that you need to tick for the CI to pass. https://github.com/rancher/charts/pull/3039#issuecomment-1735931175

Reply:
done!